wtxmgr/db.go

```diff
@@ -70,14 +151,14 @@ var byteOrder = binary.BigEndian
 // change.
 const (
 	// LatestVersion is the most recent store version.
-	LatestVersion = 1
+	LatestVersion = 2
 )
 
 // This package makes assumptions that the width of a chainhash.Hash is always 32
 // bytes.  If this is ever changed (unlikely for bitcoin, possible for alts),
 // offsets have to be rewritten.  Use a compile-time assertion that this
 // assumption holds true.
-var _ [32]byte = chainhash.Hash{}
+var _ [chainhash.HashSize]byte = chainhash.Hash{}
```

Comment:
This needs to be reverted.  It used a hardcoded 32 because it needed an assertion that the hash was actually 32 bytes.  This assertion is meaningless.